### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by deavmi**

### CODE REVIEW

File: boagen.sh [hunk 1]
```diff
@@ -39,49 +42,49 @@
 ##Begin HTML Declartion Top Area
 
 	echo "<!-- Built with BoaGen v"$VERSION" (https://github.com/deavmi/BoaGen) (The Crowbar Development Team - https://deavmi.github.io/NuCrowbar/main-site) -->" > $(cat outputf.boa)
-	echo $(cat $(cat outputf.boa))"<html>" > $(cat outputf.boa)
-	echo $(cat $(cat outputf.boa))"<head><title>"$(cat title.boa)"</title>"$(cat extra-head.boa)$(cat css.boa)"</head>" > $(cat outputf.boa)
-	echo $(cat $(cat outputf.boa))"<body bgcolor="$(cat bgcolor.boa)">" > $(cat outputf.boa)
-	echo $(cat $(cat outputf.boa))"<center>" > $(cat outputf.boa)
+	cat $OUTPUTF"\n<html>" > $OUTPUTF
```
Comment (Walkman100) [inline boagen.sh]: Does this work? just appending the text? I'd think you still need to have the echo bit, e.g.
```bash
echo $(cat $OUTPUTF)"<html>" > $OUTPUTF
```

Comment (deavmi) [inline boagen.sh]: Nope. What I have done is correct. I think.

Comment (deavmi) [inline boagen.sh]: There might be an issue with this ass there are extra arguments co cared to the cat command line args. So I think I might need to revert. :)

**Comment by deavmi** [comment]
all fixed and reverted now :)

**Comment by Walkman100** [comment]
And tested?

**Comment by deavmi** [comment]
Nope. Will test it soon tho.

**Pull request merged**